models/education-nationale.ts

```diff
@@ -0,0 +1,58 @@
+import { clientEducationNationale } from '#clients/education-nationale';
+import { HttpNotFound } from '#clients/exceptions';
+import { clientRGE } from '#clients/rge';
+import { EAdministration } from '#models/administrations';
+import {
+  APINotRespondingFactory,
+  IAPINotRespondingError,
+} from '#models/api-not-responding';
+import { Siren, verifySiren } from '#utils/helpers';
+import logErrorInSentry from '#utils/sentry';
+import { getUniteLegaleFromSlug } from './unite-legale';
+
+export interface IEducationNationaleEtablissement {
```

Comment:
This typing is not ideal : resulting type is `undefined` | `string` | `null`

I would prefer `string` and force the add of a default value on every field in the client.

This is much more resilient